src/h5web/providers/hsds/api.ts

```diff
@@ -112,13 +125,27 @@ export class HsdsApi implements ProviderAPI {
 
   /* Processing methods to fetch links and attributes in addition to the entity. Also updates API members. */
 
+  private async processAttributes(
+    collection: HDF5Collection,
+    id: HDF5Id
+  ): Promise<HDF5Attribute[]> {
+    const { attributes: bareAttributes } = await this.fetchAttributes(
```

Comment:
Perhaps we should merge my PR first, I did a small refactoring so that the fetch methods return directly the data that is relevant (in this case, `data.attributes`, instead of just `data`)